src/mito2/src/sst/parquet/row_group.rs

```diff
@@ -189,6 +196,63 @@ impl<'a> InMemoryRowGroup<'a> {
         Ok(())
     }
 
+    /// Fetches data from object store.
+    /// If the object store supports blocking, use sequence blocking read.
+    /// Otherwise, use concurrent read.
+    async fn get_byte_ranges(&self, ranges: &[Range<usize>]) -> Result<Vec<Bytes>> {
```

Comment:
We can refactor this function to a helper function that fetches ranges from an object store instead of a method of `InMemoryRowGroup`.